gamemodes/terrortown/entities/weapons/weapon_zm_molotov.lua

```diff
@@ -1,7 +1,85 @@
+---
+-- @class SWEP
+-- @section weapon_zm_molotov
+
 if SERVER then
 	AddCSLuaFile()
 end
 
+local flags = {FCVAR_NOTIFY, FCVAR_ARCHIVE, FCVAR_REPLICATED}
+
+---
+-- @realm server
+CreateConVar("ttt_firegrenade_proj_throw_force", "1", flags)
+
+---
+-- @realm server
+CreateConVar("ttt_firegrenade_proj_detonate_timer", "5", flags)
+
+---
+-- @realm server
+CreateConVar("ttt_firegrenade_proj_disarm_hit_players", "1", flags)
+
+---
+-- @realm server
+CreateConVar("ttt_firegrenade_proj_damage_trigger_behavior", "1", flags)
+
+---
+-- @realm server
+CreateConVar("ttt_firegrenade_proj_damage_trigger_instant_explosion", "0", flags)
+
+---
+-- @realm server
+CreateConVar("ttt_firegrenade_proj_damage_trigger_projectile_explosion", "1", flags)
+
+---
+-- @realm server
+CreateConVar("ttt_firegrenade_proj_prevent_damage_chain_reaction", "1", flags)
+
+---
+-- @realm server
+CreateConVar("ttt_firegrenade_proj_use_animation_timers", "0", flags)
+
+---
+-- @realm server
+CreateConVar("ttt_firegrenade_proj_spherical_physics_radius", "0", flags)
+
+---
+-- @realm server
+CreateConVar("ttt_firegrenade_proj_explosion_radius", "256", flags)
+
+---
+-- @realm server
+CreateConVar("ttt_firegrenade_proj_explosion_damage", "25", flags)
+
+---
+-- @realm server
+CreateConVar("ttt_firegrenade_proj_fire_num", "10", flags)
+
+---
+-- @realm server
+CreateConVar("ttt_firegrenade_proj_fire_size", "128", flags)
```

Comment:
one of these should also go. They essentially do the same thing: configuring how big the area of effect is. The latter is probably easier to grasp.

Reply:
The fire size is actually worse off due to an upstream source change, I did a lot of reading the plain C source of fires in order to tune it to its current number more in line with its hitbox, and, several different times Valve clobbered the original factors that controlled fires. There's an effective max size which controls its height and dimensions. If I had to cut one, it'd be the size because now with the trails for the pending embers, they're a lot easier to grasp in higher quantities.